unity-client/Assets/UnityGLTF/Scripts/Editor/GLTFImporter.cs

```diff
@@ -508,6 +511,9 @@ private GameObject CreateGLTFScene(string projectFilePath)
                 loader.useMaterialTransition = false;
                 loader.maximumLod = _maximumLod;
                 loader.isMultithreaded = true;
+                loader.importSkeleton = !_importSkeleton;
```

Comment:
Why are we negating this bool? If this is needed, we should invert the logic to avoid confusions here.

Reply:
that was actually from your original code XD. I'll check why it's that way anyway

Reply:
I removed it and tested the pipeline again, it wasn't needed at all.